src/gt4py/next/field_utils.py

```diff
@@ -0,0 +1,20 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from gt4py.next import common, utils
+
+
+@utils.get_common_tuple_value
+def get_domain(field: common.Field) -> common.Domain:
```

Comment:
This function is only used in two places and in one place it is probably wrongly used. If indeed so there is only one place left and I would remove it again. Seems a little much to have an entire module for this.

Reply:
I moved another function from an earlier PR